plugin/litedfm.vim

```diff
@@ -5,6 +5,7 @@ let s:number_default = &number
 let s:foldcolumn_default = &foldcolumn
 let s:numberwidth_default = &numberwidth
 let s:gitgutter_default = get(g:, 'gitgutter_enabled', 0)
+let s:stal_default = &showtabline
```

Comment:
Could you rename this to `tabline_default`? A bit hard at a glance to see what `stal` means.